Nosach/web-quiz-project/src/main/java/com/nosach/quizproject/filters/LoggerFilter.java

```diff
@@ -22,6 +22,7 @@ public void init(FilterConfig filterConfig) throws ServletException {
     public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
 
         HttpServletRequest req = (HttpServletRequest) servletRequest;
+        HttpServletResponse res = (HttpServletResponse) servletResponse;
```

Comment:
You are using response?

Reply:
No, I had an idea to log response headers, but then change my mind and forot to delete. Will do.